Corona-Warn-App/src/device/java/de.rki.coronawarnapp/ui/main/MainFragment.kt

```diff
@@ -68,6 +74,19 @@ class MainFragment : Fragment() {
         showOneTimeTracingExplanationDialog()
     }
 
+    override fun onActivityCreated(savedInstanceState: Bundle?) {
+        super.onActivityCreated(savedInstanceState)
+
+        if (errorResetTool.isResetNoticeToBeShown) {
```

Comment:
How about moving this block to a dedicated function "showResetNoticeIfNeeded"

Reply:
<personal opinion>
I don't like methods that are only called from one location. Use doc to make more readable.
</personal opinion>

Reply:
> How about moving this block to a dedicated function "showResetNoticeIfNeeded"

I don't like methods that are only called from one location. Use doc to make more readable.

Reply:
I concur with @chris-cwa here. I also think `errorResetTool.isResetNoticeToBeShown` is descriptive enough that no comment is warranted. But let's revisit the MainFragment for refactoring in one of the next sprints in any case!